.github/workflows/release.yaml

```diff
@@ -0,0 +1,121 @@
+
+jobs:
+  prepare-version:
+    runs-on: ubuntu-20.04
+    steps:
+      - uses: actions/create-github-app-token@v1
+        id: app-token
+        with:
+          app-id: ${{ vars.ACTIONS_APP_ID }}
+          private-key: ${{ secrets.ACTIONS_APP_PRIVATE_KEY }}
+      - name: Checkout
+        uses: actions/checkout@v4
+        with:
+          ref: ${{inputs.revision}}
+          token: ${{ steps.app-token.outputs.token }}
+      - name: Install semver tool
+        run: |
+          curl --fail -LO https://raw.githubusercontent.com/fsaintjacques/semver-tool/3.3.0/src/semver
+          chmod +x ./semver
+      - name: Check hotfix releases
+        env:
+          SEMVER_TAG: ${{ inputs.tag }}
+        run: |
+          if [[ "$SEMVER_TAG" =~ ^([[:digit:]]+\.[[:digit:]]+\.[[:digit:]]+)\.([[:digit:]]+)(.*) ]]; then
+            echo "Hotfix version detected: ${BASH_REMATCH[1]}, hotfix: ${BASH_REMATCH[2]}, prerelease: ${BASH_REMATCH[3]}"
+            echo HOTFIX_VERSION="${BASH_REMATCH[2]}" >> $GITHUB_ENV
+            echo NEW_VERSION_SUFFIX=".${BASH_REMATCH[2]}${BASH_REMATCH[3]}" >> $GITHUB_ENV
+
+            # if a hotfix version is detected, we 'tweak' the tag to make it a valid semver tag
+            # This will only be usd for internal validation, the "official" tag will be the 4-digit
+            # tag that was provided
+            SEMVER_TAG="${BASH_REMATCH[1]}-hotfix.${BASH_REMATCH[2]}${BASH_REMATCH[3]}"
+          fi
+
+          echo SEMVER_TAG="$SEMVER_TAG" >> $GITHUB_ENV
+      - name: Validate input tag
+        run: ./semver validate $SEMVER_TAG
+      - name: Prevent major/minor version change
+        run: |
+          source VERSION
+          diff=$(./semver diff "$VERSION_MAJOR.$VERSION_MINOR.$VERSION_PATCH$VERSION_SUFFIX" $SEMVER_TAG)
+          if [[ "$diff" == "prerelease" ]]
+          then
+            echo "version match"
+            exit 0
+          fi
+
+          # prevent changes on major/minor/patch version
+          # major/minor these are done manually by creating a new dev/* branch
+          # patch is done at the end of this workflow
+          echo "version mismatch, can only upgrade patch version"
+          exit 1
+      - name: Check version is on the right branch
+        run: |
+          source VERSION
+
+          # Need to unshallow and fetch dev branches first
+          git fetch --unshallow --no-tags origin HEAD '+refs/heads/development/*:refs/remotes/origin/development/*'
+
+          GIT_REMOTE=$(git remote)
+          BASE_BRANCH=$(git branch -a --list ${GIT_REMOTE}/development/${VERSION_MAJOR}.${VERSION_MINOR})
+          if ! git merge-base --is-ancestor HEAD ${BASE_BRANCH}; then
+            echo "Version commit is not included in base branch ${BASE_BRANCH}"
+            exit 1
+          fi
+
+          NEXT_MAJOR=$((VERSION_MAJOR+1)).0
+          NEXT_MINOR=${VERSION_MAJOR}.$((VERSION_MINOR+1))
+          NEXT_PATCH=${VERSION_MAJOR}.${VERSION_MINOR}.$((VERSION_PATCH+1))
```

Comment:
THis kind of branch does not exists here

Reply:
Removed the entire block.